types/three/OTHER_FILES.txt

```diff
@@ -52,7 +51,6 @@ examples/jsm/loaders/FBXLoader.d.ts
 examples/jsm/loaders/GCodeLoader.d.ts
 examples/jsm/loaders/KMZLoader.d.ts
 examples/jsm/loaders/KTXLoader.d.ts
-examples/jsm/loaders/LDrawLoader.d.ts
```

Comment:
why was this file removed from the list, and why wasn't `examples/jsm/utils/LDrawUtils.d.ts` added to the list?

Reply:
Because there's now a test file for it. That imports both the files you mentioned.

Therefore we don't need to add them to the list afaik.

We're slowly wanting to get rid of the list completely, but it's too big a task to sit down and do.